### ISSUE TITLE
bug: `ionic build ios` not working on Mac OS 10.11

### ISSUE THREAD

**Issue by stefankp** [driftyco/ionic#4414]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>ios</span> <span ionic-platform-version>9</span>  <span ionic-webview>webview</span>

<span ionic-description>`ionic build ios` does not work anymore, therefore `ionic run ios` and `ionic emulate ios` also don't work, meaning I cannot run the app through the ionic CLI. The command runs for a few seconds and then exits without any logs or error messages.

 `ionic build android` still works fine. Building and deploying the app to the simulator or phone works through Xcode. 

I am on Mac OS 10.11.1, with Xcode 7 and a complete fresh ionic install.</span>

<span is-issue-template></span>

**Comment by mpm** [comment]
I have the same issue :D according to the cordova docs, the `build` task runs two commands, `prepare` and `compile`. The one that breaks (returns without any error message) is 

`ionic prepare ios`

while

`ionic compile ios`

compiles the stuff that is there (without `prepare`, the plugins are missing though). I'm trying to understand what the `prepare` command is doing and it must be [https://github.com/apache/cordova-ios/](somewhere in this repository IMHO), but I couldn't locate it yet.

**Comment by mpm** [comment]
@stefankp and me tracked it down to [this line in the xcode npm package (v0.8.0)](https://github.com/alunny/node-xcode/blob/master/lib/pbxProject.js#L25). I don't fully understand the fork magic going on there, but the worker object never receives the `message` message (or whatever is this).

Therefore it can never trigger the `error` neither the `end` message, which would finally execute the function in `cb` (and this function will resolve a promise that resolves a ton of other promises that are currently preventing the `ionic prepare build` command from succeeding.

So, if anybody knows a bit more about the fork magic going in on the xcode package, please explain where to dig down- where is the `message` event triggert?

**Comment by mpm** [comment]
Alright, we digged down right to the bottom and the short version is:

This issue got [fixed in the node-xcode package](https://github.com/alunny/node-xcode/commit/60b7f9ec186e1eef82d0929c40461c95bb384bb1).

Ionic depends on xcode 0.6.7, so this should be bumped to 0.8.0 (but the 0.8.0 version we got earlier didn't contain this patch yet).

Anyway, everybody having the same issue, just open `parseJob.js` in your local npm repository and delete the `process.exit()` in Line 12.

We will post more helpful and proper instructions after we recovered from this exhausting investigation.
